README.md

```diff
@@ -120,112 +106,142 @@ Exemple d'une commande test en dev sur chrome pour un fichier test particulier:
 ./pc test-cafe-pro -b chrome:headless -f signup.js
 ```
 
-### Emuler avec Cordova
 
+
+## Deploy
+
+### FRONTEND WEB
+
+Pour déployer une nouvelle version web, par exemple en staging:
+**(Attention de ne pas déployer sur la prod sans authorisation !)**
```

Comment:
*autorisation

Reply:
je suis tellement bilingue que j'oublie how to properly write french words you know :p